### ISSUE TITLE
Not removing the decimal separator in suffixed inputs

### ISSUE THREAD

**Issue by eduardo-mior** [RobinHerbots/Inputmask#2511]
Sorry for my English. I am Brazilian.
I am creating a number input.
I am using alias "decimal", and suffix "%".
For some reason when I type "." and I click out of the input it keeps the "."
If I remove the suffix "%" and do the same thing as "." disappears when I click outside the input.
  
Without the suffix:
```javascript
$("#percentInput").inputmask( {
    alias: "decimal",
}); 
```
![image](https://user-images.githubusercontent.com/32434418/120220135-aba74c80-c212-11eb-9361-3dfd145f04d7.png)
  ###
  
 
With the suffix (bug):
```javascript
$("#percentInput").inputmask( {
    alias: "decimal",
    suffix: "%"
}); 
```
![image](https://user-images.githubusercontent.com/32434418/120220200-c7125780-c212-11eb-9039-3f4b3d1df795.png)

###
If my input has a suffix it keeps the decimal separator (radixPoint) when I click outside the input. If my input does not have a suffix, the decimal separator will disappear when I click away.
###
- OS: Any
- Browser: Any
- Inputmask version **5.0.6-beta.62**

**Comment by eduardo-mior** [comment]
Any prevision for this?

**Comment by eduardo-mior** [comment]
@RobinHerbots 
Is there any provision for this to be fixed? I'm not being boring, I just want to know if this is too hard to fix.


From what I've seen over the last few days you've made several bug fixes and improvements.

**Comment by RobinHerbots** [comment]
Hi @eduardo-mior ,

I will fix this in the coming days.  Sorry that I didn't give response before.

**Comment by RobinHerbots** [comment]
@eduardo-mior ,

This should be fixed in 5.0.7-beta.23.

Can you have a try!

Best regards,
Robin

**Comment by eduardo-mior** [comment]
@RobinHerbots I took my tests today. It works perfectly! Thank you very much.
